Source/Core/InputCommon/ControllerInterface/DInput/DInput.h

```diff
@@ -9,12 +9,22 @@
 #include <windows.h>
 #include <list>
 
+// Disable warning C4265 in wrl/client.h:
+//   'Microsoft::WRL::Details::RemoveIUnknownBase<T>': class has virtual functions,
+//   but destructor is not virtual
+#pragma warning(push)
+#pragma warning(disable : 4265)
+#include <wrl/client.h>
+#pragma warning(pop)
```

Comment:
This 7-line snippet keeps popping up in all the other PRs, would there be any benefit in making a dedicated header just for this ComPtr include?